statie.yaml

```diff
@@ -12,7 +12,7 @@ parameters:
         avatar: 'bio-photo.jpg'
         bio: 'Web developer'
         email: 'tomas.fejfar@gmail.com'
-        disqus-shortname: 'tomasfejfar'
+        disqusShortname: 'tomasfejfar'
```

Comment:
`-` is ignored somehow or what happened? 

I created a command to handle migration, so it should cover this.